### PR TITLE
Fixes eslint issue - a tag target 

### PR DESCRIPTION
**Pull request by ramaraz**

### CODE REVIEW

**Comment by dhbalaji** [comment]
Give a relevant PR title

File: .github/ci.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Deploy to GitHub Pages
+on:
+  push:
+    branches:
+      - develop
```
Comment (dhbalaji) [inline .github/ci.yml]: This file should not be part of this PR

Comment (ramaraz) [reply .github/ci.yml]: Its fixed
